api/api/src/main/scala/uk/ac/wellcome/platform/api/models/SearchQuery.scala

```diff
@@ -0,0 +1,11 @@
+package uk.ac.wellcome.platform.api.models
+
+case class SearchQuery(query: String, queryType: SearchQueryType)
+object SearchQuery {
+  def apply(query: String,
+            maybeQueryType: Option[SearchQueryType]): SearchQuery =
+    SearchQuery(query, maybeQueryType.getOrElse(SearchQueryType.default))
+
+  def apply(query: String): SearchQuery =
+    SearchQuery(query, None)
```

Comment:
Is this overloaded method necessary? `maybeQueryType` could be set to default of `None` above

Reply:
Or could just use `SearchQueryType.default` as the default instead of an `Option[_]`

Reply:
I overloaded this for the tests, but didn't want to make it available to the application. Will have another look to see what exactly I am trying to do here.